pages/orders.js

```diff
@@ -0,0 +1,135 @@
+import Layout from '../components/Layout'
+import useSWR from 'swr'
+import cookie from 'js-cookie';
+
+const fetcher = (url,token) => fetch(url,{headers: {'Content-Type': "application/json", 'Authorization': token}}).then((res) => res.json())
+
+const token = cookie.get("firebaseToken")
+
+class PackingOrders extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+          orders: props.data      
+        }
+    }
+
+    // Permanently deletes order
+    deleteOrder(orderId) {
+      fetch("/api/orders/DeleteOrder", {
+        method: "POST",
+        body: JSON.stringify({
+          orderId: orderId,
+        }),
+        headers: { "Content-Type": "application/json", Authorization: token },
+      }).then(() => {
+        for (var i = 0; i < this.state.orders.length; i++) {
+          if (this.state.orders[i].id == orderId) {
+            this.state.orders.splice(i, 1) // remove order from array
```

Comment:
I thought state can't be mutated, does this directly mutate state?

Reply:
Yeah, thanks for catching that! Changed it to a filter so the state doesn't get mutated directly.